### PR TITLE
Update BrickColor.yaml

### PR DESCRIPTION
**Pull request by RoloTheDevBunny**
Example used RGB values 0-255, when BrickColor takes Color3 values 0-1 and constructs the nearest BrickColor.

## Changes
Switched the colors from an RGB byte to a decimal between 0-1 and slightly modified the comment(wasn't sure how to word it).

image
As you can see, I am creating a new BrickColor with the values (255, 2, 2), which you would expect to be converted into a quite red color. Instead, these are interpreted as 0-1 values. Because the closest BrickColor to these values is Institutional white, that is the BrickColor applied.

image
Whereas if I divide all the values by 255, I get the expected result.
## Checks

By submitting your pull request for review, you agree to the following:

- [X] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [X] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [X] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 1]
```diff
@@ -14,8 +14,8 @@ description: |
   workspace.Part.BrickColor = BrickColor.new("Pastel Blue")
   -- By numerical index
   workspace.Part.BrickColor = BrickColor.new(11)
-  -- By RGB values
-  workspace.Part.BrickColor = BrickColor.new(128, 187, 219)
+  -- By RGB Color3 values 0-1
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:17]:
```suggestion
  -- By RGB decimal values in range of 0-1
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@ description: |
   workspace.Part.BrickColor = BrickColor.new("Pastel Blue")
   -- By numerical index
   workspace.Part.BrickColor = BrickColor.new(11)
-  -- By RGB values
-  workspace.Part.BrickColor = BrickColor.new(128, 187, 219)
+  -- By RGB decimal values in range of 0-1
+  workspace.Part.BrickColor = BrickColor.new(0.502, 0.733, 0.859)
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:18]:
```suggestion
  workspace.Part.BrickColor = BrickColor.new(0.502, 0.734, 0.859)
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 3]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>148</td>
-    <td>[87, 88, 87]</td>
+    <td>[0.342, 0.346, 0.342]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#161D32"}}></td>
     <td>Black metallic</td>
     <td>149</td>
-    <td>[22, 29, 50]</td>
+    <td>[0.87, 0.114, 0.197]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:463]:
```suggestion
    <td>[0.087, 0.114, 0.197]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 4]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>302</td>
-    <td>[91, 93, 105]</td>
+    <td>[0.357, 0.365, 0.412]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#0010B0"}}></td>
     <td>Dark blue</td>
     <td>303</td>
-    <td>[0, 16, 176]</td>
+    <td>[0.000, 0.63, 0.691]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:721]:
```suggestion
    <td>[0.000, 0.063, 0.691]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 5]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>329</td>
-    <td>[152, 194, 219]</td>
+    <td>[0.597, 0.761, 0.859]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#FF98DC"}}></td>
     <td>Carnation pink</td>
     <td>330</td>
-    <td>[255, 152, 220]</td>
+    <td>[0.1000, 0.597, 0.863]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:877]:
```suggestion
    <td>[1.000, 0.597, 0.863]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 6]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
   </tr>
   <tr>
     <td style={{backgroundColor: "#FF5959"}}></td>
     <td>Persimmon</td>
     <td>331</td>
-    <td>[255, 89, 89]</td>
+    <td>[0.1000, 0.350, 0.350]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:883]:
```suggestion
    <td>[1.000, 0.350, 0.350]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 7]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>336</td>
-    <td>[199, 212, 228]</td>
+    <td>[0.781, 0.832, 0.895]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#FF9494"}}></td>
     <td>Salmon</td>
     <td>337</td>
-    <td>[255, 148, 148]</td>
+    <td>[0.1000, 0.581, 0.581]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:919]:
```suggestion
    <td>[1.000, 0.581, 0.581]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 8]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>359</td>
-    <td>[175, 148, 131]</td>
+    <td>[0.687, 0.581, 0.514]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#966766"}}></td>
     <td>Copper</td>
     <td>360</td>
-    <td>[150, 103, 102]</td>
+    <td>[ 0.589, 0.404, 0.400]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:1057]:
```suggestion
    <td>[0.589, 0.404, 0.400]</td>
```

File: content/en-us/reference/engine/datatypes/BrickColor.yaml [hunk 9]
```diff
@@ -34,1249 +34,1249 @@ description: |
...
     <td>1017</td>
-    <td>[255, 175, 0]</td>
+    <td>[1.000, 0.687, 0.000]</td>
   </tr>
   <tr>
     <td style={{backgroundColor: "#12EED4"}}></td>
     <td>Teal</td>
     <td>1018</td>
-    <td>[18, 238, 212]</td>
+    <td>[0.71, 0.934, 0.832]</td>
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/datatypes/BrickColor.yaml:1195]:
```suggestion
    <td>[0.071, 0.934, 0.832]</td>
```

**Review by IgnisRBX — APPROVED** [review]

**Comment by IgnisRBX** [comment]
Thanks for catching this! I took the effort to change all of the 0-255 ranges in the table to 0-1 decimal ranges as well.

**Comment by RoloTheDevBunny** [comment]
Glad I could help! I saw the list but thought it would be a nightmare to change every BrickColor. Hope it wasn't too tedious!

**Pull request merged**